data/virt_autotest/fetch_logs_from_guest.sh

```diff
@@ -49,8 +49,8 @@ function fetch_logs_from_guest_via_ssh() {
         local logs_folder=$3
         local guest_user="root"
         local guest_pass="novell"
-        local sshpass_scp_cmd="sshpass -p ${guest_pass} scp -r ${guest_user}@${guest_ipaddr}"
-        local sshpass_ssh_cmd="sshpass -p ${guest_pass} ssh ${guest_user}@${guest_ipaddr}"
+        local sshpass_scp_cmd="sshpass -p ${guest_pass} scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -r ${guest_user}@${guest_ipaddr}"
+        local sshpass_ssh_cmd="sshpass -p ${guest_pass} ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no ${guest_user}@${guest_ipaddr}"
```

Comment:
Seems same option for multiple commands. Will it be better to save it to a arg like sshpass_shared_option and be added to all needed commands?

Reply:
sshpass_scp_cmd and sshpass_ssh_cmd are already common options. They are already shared and used by other actual ssh and scp commands. I did not add them into all ssh commands one by one, I added them into sshpass_scp_cmd and sshpass_ssh_cmd